### DISCUSSION TITLE
Sharing RX with two TXs

### DISCUSSION THREAD

**Discussion by ziiw** [ExpressLRS/ExpressLRS#1218] [Ideas]
Hi 👋 
Would it be possible hypothetically to share a RX with two different TXs?

The use case is: you have a model with an ELRS RX mounted and you sometimes want to share it with someone that also have an ELRS setup with a TX that as (obviously) a different pass phrase.

Would it be a feasible feature? 
Is it already doable with some black magic?

Thanks!

**Comment by Pelado-Mat** [comment]
I think it's related with #1333
